### PR TITLE
[Optimization][admin] Lack of H2 dependency

### PR DESCRIPTION
**Pull request by zhangyongtian**
## Purpose of the pull request
Report null pointer exception when starting directly
d0ec2d8d46b99a6e5e7d7a06c55474b

## Brief change log
Added dependency on H2

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

### CODE REVIEW

File: dinky-admin/src/main/resources/application.properties [hunk 1]
```diff
@@ -16,61 +16,63 @@
...
-#连接池配置(通常来说，只需要修改initialSize、minIdle、maxActive
+##############################################################################################################################################
+################################################# Application Level General Configuration ####################################################
+##############################################################################################################################################
+##### Druid Configuration
+# Connection pool configuration (usually, only need to modify initialSize, minIdle, maxActive)
 spring.datasource.druid.initial-size=10
 spring.datasource.druid.max-active=500
 spring.datasource.druid.min-idle=10
-# 配置获取连接等待超时的时间
```
Comment (zhangyongtian) [inline dinky-admin/src/main/resources/application.properties:25]: 47cae07f6dec9132f0b478f7544a2f1
When I read Chinese, there are garbled characters

Comment (Zzm0809) [reply dinky-admin/src/main/resources/application.properties:25]: image

如果是中文勾选图中选项

Comment (zhangyongtian) [reply dinky-admin/src/main/resources/application.properties:25]: ok收到:smiley:，看到中文打开就是乱码来着感觉有点不爽就修改修改的了，这块改不改最近？不知道翻译的有没有毛病,hh

Comment (Zzm0809) [reply dinky-admin/src/main/resources/application.properties:25]: > ok收到😃，看到中文打开就是乱码来着感觉有点不爽就修改修改的了，这块改不改最近？不知道翻译的有没有毛病,hh

very ok

**Review by Zzm0809 — APPROVED** [review]
LGTM

**Pull request merged**